### DISCUSSION TITLE
Combine solution of service & marketing cloud

### DISCUSSION THREAD

**Discussion by PremSolanki** [SalesforceCommerceCloud/sfcc-ci#407] [Team Posts]
Hi All

I'm trying to implement the combine solution of service & marketing cloud with commerce cloud.
Is there any existing code available for the same ( just to minimize the manual effort of merging the code of service-cloud-connector and marketing-cloud-connector ) ?

**Comment by pravngaur** [comment]
No, this needs to be done for each implementation. As the cartridges are
under continuous development, putting something such would quickly be
obsolete.
